tools/parse-release-notes

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { GITKEEP_FILE, UNRELEASED_DIR, RELEASE_NOTES_DIR } = require('./constants')
+const { program } = require('commander')
+const yaml = require('js-yaml')
+
+const DEFAULT_LANGUAGE = 'en'
+
+const DEFAULT_NOTE_EN = 'We’ve been working hard behind the scenes to make sure everything is working as it should. If you notice anything that does not work, let us know!\n'
```

Comment:
Should we add here an array of messages out of which we randomly select one?

Reply:
Yep, but probably better to do this in a separate issue. I'll create one.